e2e/playwright/tests/meditor.spec.ts

```diff
@@ -0,0 +1,506 @@
+import { test, expect, type Page } from "@playwright/test";
+
+const clientUrl = "http://localhost:8085";
+
+const noLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(page.getByText("LicenseNo less than 1 items")).toBeVisible();
+};
+const noContributorSelector = async (page: Page) => {
+  await expect(
+    page.getByRole("tab", { name: "Dandiset contributors Badge" })
+  ).toBeVisible();
+};
+const invalidLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(
+    page.getByRole("button", { name: "License spdx:CC-BY-NC-" })
+  ).toBeVisible();
+};
+const titleTooLongSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(
+    page.getByText("Dandiset title150 characters maximum")
+  ).toBeVisible();
+};
+
+const allDandisets = [
```

Comment:
I'm assuming all these Dandisets' metadatas are loaded into the database at some point?